### ISSUE TITLE
Schema Autocomplete does not work in conditions (i.e. WHERE)

### ISSUE THREAD

**Issue by ofirbb** [psequel/psequel-issues#52]
When writing a query there's the autocomplete feature for table and column names, however when you get to the condition clause of the query it stops working.
Also it does not work in more complex select statements such as "count(distinct COLUMN_NAME)" where it will not autocomplete as well.

**Comment by psequel** [comment]
Yeah, the autocompletion feature is very preliminary. I could not find a good SQL parser in Swift/Objective-C. Writing one from scratch will take a long time.

**Comment by thorek** [comment]
I think it's more handling of SQL Metadata than a requirement for SQL parsing. Would be happy to help if you want to :-)

**Comment by psequel** [comment]
But we need a parser first in order to provide context for autocomplete, right? Currently PSequel implements a lexer only (for syntax highlighting).
